README.md

```diff
@@ -108,7 +108,9 @@ npm run build -- Debug
 ```
 NOTE: the build will take a while to complete. Depending on your processor and memory, it could potentially take a few hours.
 
-Brave staff may also want to try [Goma](https://github.com/brave/devops/wiki/Faster-browser-builds#goma) for faster builds.
+for faster builds see: 
```

Comment:
Have discussed with the team this whole section should just go out as what we have internally is not relevant to the public.